### ISSUE TITLE
Add ALT text to images on homepage

### ISSUE THREAD

**Issue by paulproteus** [openhatch/oh-mainline#1446]
Per http://wave.webaim.org/report#/https://openhatch.org/ , there a number of accessibility issues with the OpenHatch front page.

(If you're new to accessibility, here's the one-sentence Wikipedia summary -- "Accessibility is the degree to which a product, device, service, or environment is available to as many people as possible" -- and here's an article with way more information: http://en.wikipedia.org/wiki/Accessibility )

Right now, the webaim report indicates the following problems with the OpenHatch front page:

* 10 missing ALT text problems

* 11 "redundant links" (click around within webaim to learn more, but it seems to me that the best fix is to give some images an ALT text of the empty string)

There is also a "Skipped heading level" problem, but I'd prefer to see that addressed in a separate pull request, since I think the fix will be more complicated.

Relevant facts:

* The default user profile photo is our mascot, Sufjan -- https://openhatch.org/wiki/Sufjan

* To get the OpenHatch code on your computer, see http://openhatch.readthedocs.org/en/latest/getting_started/installation.html

* The front page is powered by the https://github.com/openhatch/oh-mainline/blob/master/mysite/base/templates/base/landing.html template file.

**Comment by ehashman** [comment]
No problem @eeshangarg, good luck on your finals. Just wanted to make sure this hadn't fallen off your radar!

**Comment by shriyanka** [comment]
Hey everyone ! I think this issue is still open as it shows up there. Can I look into it?

**Comment by paulproteus** [comment]
@eeshangarg is it OK to let @shriyanka work on this? It seems like a great starter issue, so I'm happy to let @shriyanka work on it if that's OK with you.

**Comment by eeshangarg** [comment]
Hi @shriyanka! Welcome to OpenHatch! :-)

Hello @paulproteus!

I am sorry it took me so long to work on this. My high school finals are going on right now and my exam preparations take almost all of my time.

I would just like to mention that a major portion of this issue has already been implemented. I made a PR about this about 18 days ago. I forgot to make a comment about the PR here, sorry about that! Here: <https://github.com/openhatch/oh-mainline/pull/1543>. I also expressed some of my doubts in the PR message. @paulproteus It would really mean a lot to me if you could take a look at it. :-)

@paulproteus It is an honor to learn from you. See you soon at PyCon 2015. :-)

Warm regards,
Eeshan Garg

**Comment by eeshangarg** [comment]
@paulproteus: One more thing, if you think that my PR regarding this <https://github.com/openhatch/oh-mainline/pull/1543> needs more work or if the this whole issue needs more work in general, I think then it would be awesome if @shriyanka can take it from there and move further with this issue. :-)

Regards,
Eeshan

**Comment by shriyanka** [comment]
@paulproteus Hello and thank you for considering me in for it. As suggested by @eeshangarg  if you find that this issue need further 'look into' thing then I would love to work on it as it would be my first step in open source world I am very excited to fix my first bug. 
@eeshangarg Hello and Yes I would really love to take it and work on it if it requires further work.


Regards,
Shriyanka

**Comment by ehashman** [comment]
Hi all! I just did a quick accessibility check of the site (you can do so yourself by visiting [this link](http://achecker.ca/checker/index.php) and typing in 'openhatch.org') and I found a few images that are still missing alt text. They're probably located in some templates that weren't touched by #1543. I'd recommend adding that and then I will mark this issue as closed.

**Comment by eeshangarg** [comment]
Hi @ehashman!

I just made a PR for this #1643. :-)

Regards,
Eeshan Garg
